### PR TITLE
update cypress and related dependencies

### PR DESCRIPTION
**Pull request by mayank99**
## Changes

follow-up to #1595, updates to cypress 13, electron 116, and any related dependencies.

tangentially related: added `-L` flag to `serve` to suppress request logs which flood the terminal.

## Testing

updated any failing test images.

## Docs

n/a

### CODE REVIEW

File: apps/storybook/cypress-visual-screenshots/baseline/Tabs.test.ts-Vertical.png
Comment (rohan-kadkol) [inline apps/storybook/cypress-visual-screenshots/baseline/Tabs.test.ts-Vertical.png:1]: To confirm, is this test image change alright?

File: package.json [hunk 1]
```diff
@@ -73,8 +73,7 @@
     "yaml": "^2.2.2",
     "semver": "^7.5.2",
     "optionator": "~0.9.3",
-    "typescript": "~5.1.6",
-    "@cypress/request": "^3.0.0"
```
Comment (rohan-kadkol) [inline package.json:77]: `@cypress/request` is no longer needed?

File: apps/storybook/cypress-visual-screenshots/baseline/Tabs.test.ts-Vertical.png
Comment (mayank99) [reply apps/storybook/cypress-visual-screenshots/baseline/Tabs.test.ts-Vertical.png:1]: yes, that's how it should look. https://itwin.github.io/iTwinUI/dev/react/iframe.html?args=&id=core-tabs--vertical&viewMode=story

File: package.json (hunk 1 above)
Comment (mayank99) [reply package.json:77]: this is an override that was previously needed to fix npm audit

**Review by rohan-kadkol — APPROVED** [review]

**Review by gretanausedaite — APPROVED** [review]

**Pull request merged**